content/spin/v2/key-value-store-tutorial.md

```diff
@@ -72,6 +73,17 @@ $ spin new -t http-ts spin-key-value
 
 {{ startTab "Python" }}
 
+## Prerequisite
```

Comment:
This interacts uncomfortably with the "we'll start from a template using the following commands:" text that introduces the tab block.  The other blocks follow on from that, by starting with the template install command.

The best way to avoid this would be to move this paragraph to the existing "Tutorial Prerequisites" section.

If you strongly wish to have it here, you could put it _after_ the `spin new` command so that the "introductory para to tab content" flow is preserved.